### DISCUSSION TITLE
Fuxa @  SPS Nuremberg

### DISCUSSION THREAD

**Discussion by robbudge** [frangoteam/FUXA#983] [Show and tell]
FYI Thats not me :)
![IMG_1657](https://github.com/frangoteam/FUXA/assets/15365138/2eb12169-c515-46f9-b20d-3e2f6e8ea1b3)
![IMG_1658](https://github.com/frangoteam/FUXA/assets/15365138/1a52eba1-5fcf-45cb-8ad8-e05d36ab49fe)

**Comment by unocelli** [comment]
Look really good, Bravo!
